Configuration/TSconfig/Page/mod.ts

```diff
@@ -14,9 +14,19 @@ mod {
                 list_type = hairu_auth
               }
             }
+            hairu_updatepw {
```

Comment:
The previous name `hairu_password` was perfectly fine.

Reply:
In one of the previous review comments you wish that i changed thise key based on "hairu_auth".

Reply:
Not sure what you mean here, previously I requested that you merge the two `addToList()` calls into one.